src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.ts

```diff
@@ -1,13 +1,27 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { SidebarCardElement, SidebarTypes } from 'app/types/sidebar';
 
 @Component({
     selector: 'jhi-sidebar-card-item',
     templateUrl: './sidebar-card-item.component.html',
     styleUrls: ['./sidebar-card-item.component.scss', '../sidebar.component.scss'],
 })
-export class SidebarCardItemComponent {
+export class SidebarCardItemComponent implements OnInit {
     @Input() sidebarItem: SidebarCardElement;
     @Input() sidebarType?: SidebarTypes;
     @Input() groupKey?: string;
+    @Input() unreadCount: number = 0;
```

Comment:
I think we should use input signals over @Input() according to the docs.

```suggestion
    unreadCount = input<number>(0);
```